docs/getting-started/quick-start.md

```
@@ -18,6 +18,10 @@ Get started with TanStack AI in minutes. This guide will walk you through creati
 
 > **Using a different framework?** See quick-starts for [Vue](./quick-start-vue), [Svelte](./quick-start-svelte), or [server-only Node.js](./quick-start-server).
 
+> **React Native or Expo app?** Use the headless React hooks with an absolute
+> server URL and a mobile-compatible transport. See
+> [Quick Start: React Native](./quick-start-react-native).
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)